src/components/@shared/AssetList/index.tsx

```diff
@@ -41,22 +44,25 @@ export default function AssetList({
     useState<AssetExtended[]>(assets)
   const [loading, setLoading] = useState<boolean>(isLoading)
   const isMounted = useIsMounted()
+  const newCancelToken = useCancelToken()
 
   useEffect(() => {
     if (!assets || !assets.length) return
 
     setAssetsWithPrices(assets as AssetExtended[])
     setLoading(false)
-    async function fetchPrices() {
-      const assetsWithPrices = await getAccessDetailsForAssets(
-        assets,
-        accountId || ''
-      )
+
+    async function fetchPrices(token?: CancelToken) {
```

Comment:
this seems like the wrong tactic. The prices coming directly from the Aquarius-delivered DDO implies we do not have to fetch them again. So this component here can finally be a pure visual component without any data fetching. That is the whole goal of getting prices from Aquarius.

Remove all fetching logic from in here and refactor to get the price from `assets`